EngineeringModel.Tests/ViewModels/FiniteStateBrowser/FiniteStateBrowserViewModelTestFixture.cs

```diff
@@ -192,8 +192,17 @@ public void VerifyThatContextMenuPopulated()
             this.iteration.ActualFiniteStateList.Add(actualList);
 
             var viewmodel = new FiniteStateBrowserViewModel(this.iteration, this.session.Object, this.thingDialogNavigationService.Object, this.panelNavigationService.Object, null, null);
-            
-            // no row selected
```

Comment:
"no row selected" scenario is not in the test anymore

Reply:
No row selected was actually selected row of one of the two options, which are being covered by the cases. It refereed to rows that are not children.

Reply:
You can still add an assertion when `SelectedThing = null`.

Reply:
agreed

Reply:
I do not think that is necessary. SelectedThing cannot be null. There are always 2 rows created in initialization. In my opinion that check is redundant.

Reply:
The check for null selection has been added.